vsts/vsts.yaml

```diff
@@ -43,9 +40,7 @@ variables:
   testNetcore31: ${{ or(eq(variables['minMatrix'], 'True'), contains(parameters['testTargets'], 'netcore3.1')) }}
 
 # The remaining build/test targets.
-  testNet50: ${{ or(eq(variables['allTestTargets'], 'True'), contains(parameters['testTargets'], 'net5.0')) }}
-  testNetcore21: ${{ or(eq(variables['allTestTargets'], 'True'), contains(parameters['testTargets'], 'netcore2.1')) }}
-  testNet472: ${{ or(eq(variables['allTestTargets'], 'True'), contains(parameters['testTargets'], 'net4.7.2')) }}
+  # Add remaining targets in here
```

Comment:
I've left this template in here for future reference

Reply:
Shall we add .net 7.0 testing?

Reply:
How about we remove net3.1 and add net 7 (net 3.1 is set to go out of support end of this year).

Reply:
k